pyproject.toml

```diff
@@ -16,7 +16,6 @@ pyink-indentation = 2
 pyink-use-majority-quotes = true
 
 [tool.pytest.ini_options]
-required_plugins = ["pytest-xdist"]
 addopts = "-n auto"
```

Comment:
revert please (and commit with `git commit --amend --no-edit` to ensure things stay squashed)